### DISCUSSION TITLE
Stylings does not apply on ultima template v9

### DISCUSSION THREAD

**Discussion by siposm17** [primefaces/community#2274] [PrimeReact Templates]
I have purchased ultima template (React) and downloaded the v9 version. Opened the typescript version, ran npm i and npm start then the project starts. There are no stylings applied as shown on the picture. However on the javascript version, following the same steps styling is just fine. There are no errors or anything in the console.
![image](https://github.com/primefaces/community/assets/74093295/7c151b18-cb20-4933-a2a2-f48a827f5f92)

**Comment by mjauditlytics** [comment]
I have a similar issue with PrimeVue 4 - trying to move from Sakai + Primvue 3
